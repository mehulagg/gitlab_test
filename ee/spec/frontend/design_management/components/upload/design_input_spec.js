import { mount } from '@vue/test-utils';
import DesignInput from 'ee/design_management/components/upload/design_input.vue';

describe('Design management upload button component', () => {
  let wrapper;

  function createComponent() {
    wrapper = mount(DesignInput);
  }

  afterEach(() => {
    wrapper.destroy();
  });

  it('renders design input', () => {
    createComponent();

    expect(wrapper.element).toMatchSnapshot();
  });
});
