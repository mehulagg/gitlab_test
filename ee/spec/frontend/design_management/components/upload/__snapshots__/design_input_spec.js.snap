// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Design management upload button component renders design input 1`] = `
<input
  accept="image/*"
  class="hide"
  multiple="multiple"
  name="design_file"
  type="file"
/>
`;
